src/platform/lunarlake/include/platform/lib/clk.h

```diff
@@ -14,7 +14,7 @@
 
 #include <ace/lib/clk.h>
 
-#define CLK_MAX_CPU_HZ		400000000
+#define CLK_MAX_CPU_HZ		CONFIG_XTENSA_CCOUNT_HZ
```

Comment:
Are you sure about the 393.2 MHz value?

It should be an integer multiple of 24.576 MHz and it's off a bit.

should be 393.216 MHz

Reply:
Yes, you are right. The exact value is `393216000` and this is given as the default for `CONFIG_XTENSA_CCOUNT_HZ`.
`393.2 MHz` is taken from documentation.